dashboard/src/reducers.js

```diff
@@ -43,7 +43,7 @@ const makeRootReducer = () => (state, action) => {
     window.location.href = '/'
   }
 
-  return combineReducers({
+  const finalState = combineReducers({
```

Comment:
Ah yes, you're correct. In the end we got rid of any of the changes made within this reducer. Will revert to the way it previously existed.